integration/core/test_cli.py

```diff
@@ -658,11 +658,15 @@ def get_backup_url(bin, backupID):
         output = subprocess.check_output(cmd).strip()
         backup = json.loads(output)
         if backup['progress'] == 100 and 'backupURL' in backup.keys():
+            assert backup['state'] == "complete"
```

Comment:
it's better if use `backup['state']` to check e.g. progress, error etc. instead of using it in a reverse way.

Same below.